File: tamagochi-laravel/app/Models/Animal_stat.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Animal_stat extends Model
{
    protected $fillable =
        [
            'user_id', 'animal_id', 'name', 'hunger', 'thirst', 'happiness',
            'activity', 'health', 'dexterity', 'action_count', 'last_hunger', 'last_thirst',
            'last_happiness', 'last_activity', 'last_health', 'last_dexterity', 'last_action'
        ];
    protected $attributes = [
        'hunger' => 100,
        'thirst' => 100,
        'happiness' => 100,
        'activity' => 100,
        'health' => 100,
        'dexterity' => 100,
        'action_count' => 10
    ];

    public $timestamps = false;

    public static function boot()
    {
        parent::boot();

        static::creating(function ($model) {
            $model->created_at = $model->freshTimestamp();
            $model->last_hunger = $model->created_at;
            $model->last_thirst = $model->created_at;
            $model->last_happiness = $model->created_at;
            $model->last_activity = $model->created_at;
            $model->last_health = $model->created_at;
            $model->last_dexterity = $model->created_at;
            $model->last_action = $model->created_at;
        });
    }

    public function animal()
    {
        return $this->hasOne(Animal::class, 'animal_id', "id",);
    }

    public function owner()
    {
        return $this->belongsTo(User::class, 'user_id', 'id');
    }
}

File: Tamagochi/Tamagochi/Model/AnimalStat.cs
using System;
using System.Collections.Generic;

namespace Tamagochi.Model
{
    public partial class AnimalStat
    {
        public ulong Id { get; set; }
        public ulong UserId { get; set; }
        public ulong AnimalId { get; set; }
        public string Name { get; set; } = null!;
        public int Hunger { get; set; }
        public int Thirst { get; set; }
        public int Happiness { get; set; }
        public int Activity { get; set; }
        public int Health { get; set; }
        public int Dexterity { get; set; }
        public DateTime LastHunger { get; set; }
        public DateTime LastThirst { get; set; }
        public DateTime LastHappiness { get; set; }
        public DateTime LastActivity { get; set; }
        public DateTime LastHealth { get; set; }
        public DateTime LastDexterity { get; set; }
        public DateTime CreatedAt { get; set; }

        public virtual Animal Animal { get; set; } = null!;
        public virtual User User { get; set; } = null!;
    }
}
